src/main/java/org/springframework/social/partnercenter/api/billing/usage/SpendingBudget.java

```diff
@@ -0,0 +1,15 @@
+package org.springframework.social.partnercenter.api.billing.usage;
+
+import java.math.BigDecimal;
+
+public class SpendingBudget {
```

Comment:
Is this related to Budget? i.e. Should this extend Budget or should Budget be an Interface that this implements.

Reply:
Your right. I saw the SpendingBudget object in the microsoft SDK but I now see I had implemented it as Budget. SpendingBudget is redundant. I'll modify the Budget class though to use BigDecimal and extend ResourceBase however to be in keeping with more recent changes